### DISCUSSION TITLE
Toggle zoom include actual size

### DISCUSSION THREAD

**Discussion by andr-groth** [sumatrapdfreader/sumatrapdf#2513] [Ideas]
Since _actual size_ appears in the list of zoom options (ctrl 0/1/2/3), it would be nice and logical to include _actual size_ also in _toggle zoom_ (z).

**Comment by GitHubRulesOK** [comment]
Hmm toggle should normally be two states  so the current tri-state is to me confusing (especially when page is the same as content), the norm would be use CTRL+1 and another
With new ability to program your own key actions you could set different keys but currently no means to toggle more than one key except via two entries say shift and or ctrl

 so you could set Shift Z to "Actual Zoom" still two keys but closer together, alternately just assign X
